.circleci/config.yml

```diff
@@ -14,7 +14,7 @@ jobs:
       - run: make format
       - run: make
       - run: make golangci-lint
-      - run: make test
```

Comment:
would it for for CirleCI if we allowed to pass `-p 1` to the existing test target?

Reply:
Hmm, I din't realize such a flag exists. Looks to be a top-level go toolchain flag and based on the description, it probably helps. I'll give it a try.

Reply:
Seems to do the trick. Thanks for pointing out, it is much less intrusive (and more general) this way.